### ISSUE TITLE
Swept paper wallets and the app has crashed.

### ISSUE THREAD

**Issue by mitchelldemeter** [bitcoin-wallet/bitcoin-wallet#321]
I am looking for some assistance with the Bitcoin App. I swept 3 paper wallets into the app on Friday. Two of the wallets seemed to work fine, one of them didn't look like it worked properly so I tried again, it still didn't seem to work so I increased the fee. The app then crashed and will not open. Is there any chance that an update will be released anytime soon to resolve this bug and allow me to open the app.

Any possible assistance would be greatly appreciated.

Kind regards,

Mitchell Demeter

Please see attached screenshot.
![bitcoinscreenshot2](https://cloud.githubusercontent.com/assets/19787480/15842005/3cca8212-2c0a-11e6-9ce2-90d2a0aec205.jpg)
![bitcoinscreenshot](https://cloud.githubusercontent.com/assets/19787480/15842006/3ccd61ee-2c0a-11e6-8a10-65b1daf78f03.jpg)

**Comment by mitchelldemeter** [comment]
@schildbach - Would you have any insight into what could be causing this issue?

Thanks

**Comment by schildbach** [comment]
Can you report the issue through the app? You've already seen the correct dialog.

**Comment by mitchelldemeter** [comment]
Absolutely, I have submitted a few bug reports though the app.

Thank you.

**Comment by schildbach** [comment]
What email address have you been using for mailing the reports?

**Comment by mitchelldemeter** [comment]
I have been emailing from mitchelldemeter@gmail.com - have you received them?

Thank you!

**Comment by schildbach** [comment]
No we didn't. Can you check the outbox and sent folders of your email app on the phone? Maybe it stil sits there unsent.

**Comment by mitchelldemeter** [comment]
This is where I am sending it from, it doesn't appear to be sending out of
my inbox?

Is there another way for me to send the report?

Thanks

**Comment by schildbach** [comment]
Did you look it up in your email app? Just try again – wait until the app crashes, then start the app again. It should show the "previous crash detected" dialog. Tap report, tap send. It should open your email app.

**Comment by mitchelldemeter** [comment]
These are screenshots of what I'm seeing. It says it has been sent.
On Jun 8, 2016 2:30 AM, "Andreas Schildbach" <notifications@github.com>
wrote:

Did you look it up in your email app? Just try again – wait until the app
crashes, then start the app again. It should show the "previous crash
detected" dialog. Tap report, tap send. It should open your email app.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/bitcoin-wallet/bitcoin-wallet/issues/321#issuecomment-224537652>,
or mute the thread
<https://github.com/notifications/unsubscribe/AS3u2N6T_4Yr2kxo7JdW49gsntlf_2d8ks5qJovKgaJpZM4IvcER>
.

**Comment by schildbach** [comment]
Ok sorry, you need to find someone locally to help you with your email issue. Maybe ask the publisher of your email app.

**Comment by mitchelldemeter** [comment]
Alright, I just figured out that it will open when I am not connected to
the Internet.  I just sent the email with the full report. Also, here is a
screenshot of the transactions that I believe caused the issue.

I swept 3 paper wallets and one didn't seem to work so I tried again and
had the same issue so I increased the fee and it all crashed.

Thank you very much for your help.

Kind regards,

Mitchell Demeter
